kolibri/core/webpack/management/commands/webpack_json.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+
+from kolibri.core.webpack.hooks import WebpackBundleHook
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    args = ()
+    help = 'Creates a new schema'  # @ReservedAssignment
+    option_list = BaseCommand.option_list + ()
+
+    def handle(self, *args, **options):
+
+        logging.debug(args)
+
+        for hook in WebpackBundleHook().registered_hooks:
```

Comment:
OK - so that means that to build a plugin's JS, it has to be enabled, yes?

We can definitely use this functionality to reduce the webpack configuration to something better (and even allow for individual plugins to define additional webpack configuration for themselves, and specify their own npm dependencies). @indirectlylit take note!